ManageWikiSettings.php

```diff
@@ -2429,6 +2429,78 @@
 			],
 		],
 	],
+	'wgShareUsePlainLinks' => [
+		'name' => 'Use plain links for sidebar share menu',
+		'from' => 'share',
+		'type' => 'check',
+		'overridedefault' => true,
+		'section' => 'socialtools',
```

Comment:
If we are adding this to "Social tools", we need to rename the section to "Social", as "Social tools" is the social tools extensions (IE SocialProfile), or configuration involving the social tools extensions.

Reply:
If you're fine with that then sure but I added it there as I really can't think of where else to put it.

Reply:
I will just do the section rename later. This can be merged as-is for now I suppose.